docs/index.md

```diff
@@ -176,6 +176,7 @@ that contains the current referrer.
 * `this.$context.locator` – the [Service Locator](#service-locator) of the application.
 * `this.$context.getRouteURI('routeName', { parameter: 'value' })` – generates a URI string pasting parameter values to the specified route expression (must have a name). If a route parameter is used as a query value in URI you can use Array as a value for it. See examples in the [Routing](#named-routes-and-uri-generation) section.
 * `this.$context.redirect('String')` – redirects to a specified location string. If used while rendering the `document` or `head` component, this action will be accomplished using HTTP headers and status codes on the server, else via an inline `<script>` tag.
+* `this.$context.permRedirect('String')` – redirects to a specified location string with 301 HTTP code status.
```

Comment:
"redirects to a specified location string with 301 HTTP code status." => "Does exactly the same as `this.$context.redirect('url')` but uses HTTP status code 301 instead of 302."